zebra/zebra_vty.c

```diff
@@ -1155,10 +1186,12 @@ DEFPY (show_route,
 	if (vrf_all) {
 		RB_FOREACH (vrf, vrf_name_head, &vrfs_by_name) {
 			struct zebra_vrf *zvrf;
-			struct route_table *table;
+			struct route_table *table = NULL;
 
-			if ((zvrf = vrf->info) == NULL
-			    || (table = zvrf->table[afi][SAFI_UNICAST]) == NULL)
+			zvrf = vrf->info;
```

Comment:
Again, please drop these style changes as they have nothing to do with the rest of the commit.